Directory.Packages.props

```diff
@@ -0,0 +1,74 @@
+<Project>
```

Comment:
Can you organize the package references by area?

Like: Microsoft.Extensions.* together, Azure.* together, Grpc / protobuf together, CodeAnalysis together, testing frameworks together, etc.